packages/quantic/force-app/main/default/lwc/quanticRefineModalContent/quanticRefineModalContent.css

```diff
@@ -0,0 +1,5 @@
+.filters-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
```

Comment:
Just FYI, I don't know what we've decided to do here, but there is already slds classes to do that afaik